roles/nginx/meta/main.yml

```diff
@@ -4,6 +4,7 @@ dependencies:
    collectd_deployed_app_name: "nginx"
    collectd_deployed_app_version: "{{ nginx_version }}"
    collectd_scripts: ["nginx"]
+   collectd_version: "{{ server_collectd_version }}"
```

Comment:
Should the value not be from `nginx_collectd_version` instead? It seems to be going against how variables are named in the role.

Reply:
updated